PandaGrom/hometask-3/rainfall.rb

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './data/rainfalls'
+require './data/towns'
+require './run'
+
+# This class calculate average and variance from town rainfalls
+class Rainfall
+  extend RunCli
+  attr_accessor :town, :strng
+
+  def initialize(town:, strng:)
+    @town = town
+    @strng = strng
+  end
+
+  def mean
+    return -1 unless find_town
+
+    average
+  end
+
+  def variance
+    return -1 unless find_town
+
+    calculating_variance
+  end
+
+  private
+
+  def find_town
+    strng.split(/\n/).find { |str| str.match?(town) }
+  end
+
+  def parsing_values
```

Comment:
call it `parsed_values` or something like this
usually, we do not call variables in this way